Resources/Private/Form/StaticText.html

```diff
@@ -0,0 +1,6 @@
+<f:if condition="{element.label}">
+    <h2>{element.label -> f:translate(id: 'forms.elements.{element.identifier}.label', package: element.renderingOptions.translationPackage) -> f:format.raw()}</h2>
+</f:if>
+<f:if condition="{element.properties.text}">
+    <p{f:if(condition: element.properties.elementClassAttribute, then: ' class="{element.properties.elementClassAttribute}"')}>{element.properties.text -> f:translate(id: 'forms.elements.{element.identifier}.text', package: '{element.renderingOptions.translationPackage}') -> f:format.nl2br() -> f:format.raw()}</p>

```

Comment:
using `format.raw()` is potentially risky (as you mentioned in the comments).
Changing
```html
{element.properties.text -> f:translate(id: 'forms.elements.{element.identifier}.text', package: '{element.renderingOptions.translationPackage}') -> f:format.nl2br()}
```

to

```html
{f:translate(id: 'forms.elements.{element.identifier}.text', package: '{element.renderingOptions.translationPackage}', value: element.properties.text) -> f:format.nl2br()}
```

in the default template should avoid the double encoding while keeping it secure